src/Core/Content/Seo/SeoUrlTwigFactory.php

```diff
@@ -45,7 +45,7 @@ public function createTwigEnvironment(SlugifyInterface $slugify, iterable $twigE
 
         $twig->getRuntime(EscaperRuntime::class)->setEscaper(
             SeoUrlGenerator::ESCAPE_SLUGIFY,
-            static fn ($string) => rawurlencode($slugify->slugify((string) $string))
+            static fn ($string) => rawurlencode($slugify->slugify($string))
```

Comment:
```suggestion
            static fn (string $string) => rawurlencode($slugify->slugify($string))
```

just to be sure? :grin:

Reply:
there are some more places, do as you like :grin:

Reply:
test also says no :grimacing:

Reply:
Ah yes, this cast cannot be removed :see_no_evil: I reverted this change.